routing/router.go

```diff
@@ -1108,7 +1141,8 @@ func (r *ChannelRouter) processUpdate(msg interface{}) error {
 				"graph: %v", msg.PubKeyBytes, err)
 		}
 
-		log.Infof("Updated vertex data for node=%x", msg.PubKeyBytes)
+		log.Tracef("Updated vertex data for node=%x", msg.PubKeyBytes)
+		r.stats.numNodesUpdates++
```

Comment:
This is called concurrently with other goroutines, so we either need a wrapper method with a mutex, or usage of an `atomic` increment and laod.

Reply:
Applies to all other spots where we increment/read the new `stats` attribute.

Reply:
good catch, fixed